galley/pkg/config/schema/metadata.yaml

```diff
@@ -51,26 +55,32 @@ collections:
   - name: "istio/mixer/v1/config/client/quotaspecs"
     kind: "QuotaSpec"
     group: "config.istio.io"
+    pilot: true
```

Comment:
seems odd to have pilot: true for a mixer v1 related spec.

Reply:
many parts of the codebase assume that only a restricted set of schemas are contained in the collection. I called it "Pilot" for now, because it's primarily the collection on schemas used by Pilot.  In the old code, it was called "Istio", but Galley's "Istio" includes everything (which is probably appropriate).